stackslib/src/chainstate/stacks/boot/signers-voting.clar

```diff
@@ -39,21 +50,25 @@
 (define-read-only (get-vote (reward-cycle uint) (round uint) (signer principal))
     (map-get? votes {reward-cycle: reward-cycle, round: round, signer: signer}))
 
+(define-read-only (get-candidate-info (reward-cycle uint) (round uint) (candidate (buff 33)))
+    {candidate-weight: (default-to u0 (map-get? tally {reward-cycle: reward-cycle, round: round, aggregate-public-key: candidate})),
+    total-weight: (map-get? cycle-total-weight reward-cycle)})
+
 (define-read-only (get-tally (reward-cycle uint) (round uint) (aggregate-public-key (buff 33)))
     (map-get? tally {reward-cycle: reward-cycle, round: round, aggregate-public-key: aggregate-public-key}))
 
-(define-read-only (get-current-signer-slots (signer-index uint))
+(define-read-only (get-current-signer-weight (signer-index uint))
     (let ((cycle (+ u1 (burn-height-to-reward-cycle burn-block-height))))
-      (get-signer-slots signer-index cycle)))
+      (get-signer-weight signer-index cycle)))
 
-(define-read-only (get-signer-slots (signer-index uint) (reward-cycle uint))
-    (let ((details (unwrap! (try! (contract-call? .signers get-signer-by-index reward-cycle signer-index)) err-invalid-signer-index)))
-        (asserts! (is-eq (get signer details) tx-sender) err-signer-index-mismatch)
+(define-read-only (get-signer-weight (signer-index uint) (reward-cycle uint))
+    (let ((details (unwrap! (try! (contract-call? .signers get-signer-by-index reward-cycle signer-index)) (err ERR_INVALID_SIGNER_INDEX))))
```

Comment:
As a user of this function, I can receive `(err u2)` and don't know whether the error came from this contract or from `signers` contract.

The error codes should not overlap.

Reply:
Good point!

Reply:
@setzeus, since you're already modifying these, can you update the error values in your PR to avoid overlap?

Reply:
@obycode @friedger thoughts on different prepended values per contract? IE:

signers: 100, 101, 102
signrs-voting: 200, 201,202